src/macaron/__main__.py

```diff
@@ -144,7 +146,20 @@ def verify_policy(verify_policy_args: argparse.Namespace) -> int:
             logger.critical('The policy file "%s" does not exist.', verify_policy_args.file)
             return os.EX_OSFILE
 
-        result = run_policy_engine(verify_policy_args.database, verify_policy_args.file)
+        with open(verify_policy_args.file, encoding="utf-8") as file:
+            policy_content = file.read()
+
+        result = run_policy_engine(verify_policy_args.database, policy_content)
+        vsa = generate_vsa(policy_content=policy_content, policy_result=result)
+        if vsa is not None:
+            vsa_filepath = os.path.join(global_config.output_path, "vsa.intoto.jsonl")
+            logger.info("Generating the Verification Summary Attestation (VSA) to %s.", vsa_filepath)
```

Comment:
Can you please also add a log to help users decode and read the VSA output?
```
            logger.info(
                "To decode and see the payload run `cat %s | jq -r '.payload' | base64 -d | jq`.",
                vsa_filepath,
            )
```

Reply:
Ah, I think I missed this in your suggestion earlier. Thanks for the reminder.